### PR TITLE
Add some doc regarding user management in the admin interface

### PR DESCRIPTION
**Pull request by kalbermattenm**
Could anybody confirm me that I have not written wrong stuff... ?

But I think that this is really missing in the documentation.

### CODE REVIEW

File: doc/administrator/administrate.rst [hunk 1]
```diff
@@ -181,3 +181,30 @@ to see (and edit, if applicable) the features within this area.
 * ``Roles``: Restricted to the following roles.
 * ``Read/write``: Allows the linked users to change the objects.
 * ``Area``: Active in the following area, if not defined it is active everywhere.
+
+Users
+-----
+
+Each user may have from 1 to n roles, but each user has a default role from
+which are taken the some settings. Currently the default role (defined through
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst]: "the some settings" -> "some settings"

File: doc/administrator/administrate.rst [hunk 2]
```diff
@@ -181,3 +181,30 @@ to see (and edit, if applicable) the features within this area.
...
+the "Settings from role" selection) has an influence on the extent retrieved
+through the role and some functionalities.
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst]: not sure I understand... to which part of the sentence does "and some functionalities." refer to?

File: doc/administrator/administrate.rst [hunk 3]
```diff
@@ -181,3 +181,30 @@ to see (and edit, if applicable) the features within this area.
...
+
+Role extents for users can only be set in one role, because the application
+is currently not able to check multiple extents for one user, thus it is the
+default role which defines this unique extent.
+
+Any functionality defined as ``single``, hence only one can be defined per
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst]: Suggestion:
Any functionality specified as ``single`` can be defined only once per user. Hence, these functionalities have to be defined in the default role.

File: doc/administrator/administrate.rst [hunk 4]
```diff
@@ -181,3 +181,30 @@ to see (and edit, if applicable) the features within this area.
...
+user, has to be defined in this default role. Any ``single`` functionality
+defined in another role will not be taken into account.
+
+By default functionalities have ``single`` not set. Currently, the following
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst]: Suggestion:
By default, functionalities are not specified as ``single``. Currently, ...

File: doc/administrator/administrate.rst (hunk 2 above)
Comment (kalbermattenm) [reply doc/administrator/administrate.rst]: Would that be clearer:

```
Currently the default role (defined through the "Settings from role" selection) has an
influence on the role extent and on some functionalities regarding their configuration.
```

File: doc/administrator/administrate.rst (hunk 3 above)
Comment (kalbermattenm) [reply doc/administrator/administrate.rst]: Agreed

File: doc/administrator/administrate.rst (hunk 2 above)
Comment (jwkaltz) [reply doc/administrator/administrate.rst]: Yes, thanks

**Comment by kalbermattenm** [comment]
Thanks a lot for the review !

**Comment by sbrunner** [comment]
@jwkaltz if it's OK for you, you can merge it :-)

**Comment by jwkaltz** [comment]
OK for me @sbrunner , but Jenkins has somehow disappeared, and is blocking the merge

**Pull request merged**